### PR TITLE
Drop W&B completely

### PR DESCRIPTION
**Pull request by yartem**

### CODE REVIEW

File: {{cookiecutter.project_slug}}/.gitignore [hunk 1]
```diff
@@ -5,7 +5,6 @@
 /config/*.json
 /config/*.yml
 /.setup_done
-/.wandb_sweeps
```
Comment (YevheniiSemendiak) [inline {{cookiecutter.project_slug}}/.gitignore:8]: Do [this](https://github.com/neuromation/cookiecutter-neuro-project/blob/ay/drop-wandb/%7B%7Bcookiecutter.project_slug%7D%7D/.gitignore#L15) guy is kept for user *wandb* usage in perspective?

Do we need [this](https://github.com/neuromation/cookiecutter-neuro-project/blob/ay/drop-wandb/.gitignore#L16) one?

Comment (mariyadavydova) [reply {{cookiecutter.project_slug}}/.gitignore:8]: I agree that we should remove both.

**Comment by anayden** [comment]
Removed all mentions of W&B mentioned by @YevheniiSemendiak

File: {{cookiecutter.project_slug}}/.gitignore (hunk 1 above)
Comment (YevheniiSemendiak) [reply {{cookiecutter.project_slug}}/.gitignore:8]: ok, got it

**Comment by YevheniiSemendiak** [comment]
lgtm, thank you, @anayden

**Review by YevheniiSemendiak — APPROVED** [review]
approve from my side

**Pull request merged**